client/src/api/lectureAPI.js

```diff
@@ -91,4 +91,33 @@ async function getLecture(id) {
     }
 }
 
-export  {getLecture, getLectures}
\ No newline at end of file
+/**
+ * Cancel a lecture
+ *
+ * @param {*} lecture_id valid lecture id
+ * @returns true on success or throws error
+ */
+async function cancelLecture(lecture_id){
+    let url = `/lectures/${lecture_id}`;
```

Comment:
As discussed above, there is an inconsistency between this url and the one that the server expects: `'/teachers/:teacher_id/lectures/:lecture_id`, atm it responds with 404